packages/amplify-graphql-auth-transformer/package.json

```
@@ -35,6 +35,7 @@
     "@aws-cdk/aws-dynamodb": "~1.124.0",
     "@aws-cdk/aws-iam": "~1.124.0",
     "@aws-cdk/core": "~1.124.0",
+    "amplify-cli-core": "^2.4.18",
```

monorepo pacakges must use exact types

```suggestion
    "amplify-cli-core": "2.4.18",
```

---

updated